.github/workflows/run_tests.yaml

```diff
@@ -0,0 +1,22 @@
+name: Run tests
+on:
+  pull_request:
+  push: { branches: main }
+
+jobs:
+  test:
+    name: Run test suite
+    runs-on: ubuntu-latest
+    container: intel/oneapi-basekit:2022.2-devel-ubuntu20.04
```

Comment:
if size is not an issue we could use our own image after all ?

Reply:
Yes, I've found the issue I add.
I will push another commit soon.